PantryNodeReact/src/pages/login.tsx

```diff
@@ -1,14 +1,116 @@
-import { Button } from "@mui/material";
-import React from "react";
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+// import FormControlLabel from '@mui/material/FormControlLabel';
+// import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
 import { useNavigate } from "react-router-dom";
-const Login = () => {
-  const navigate = useNavigate();
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+function Copyright(props: any) {
   return (
-    <div>
-      login
-      <Button onClick={() => navigate("/")}> Submit</Button>
-    </div>
+    <Typography variant="body2" color="text.secondary" align="center" {...props}>
+      {'Copyright © '}
+      
+        Pantry Node
+      {' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
   );
-};
+}
```

Comment:
Can we have this component in a separate file under Components folder since we are using it twice again in signup page?

Reply:
Sure that is something can be done later I wanted to get this merged today so we could talk to the backend team about having them then provide the endpoints for authentication.

Reply:
I did this one

Reply:
I did all of these approve this